Spanner/src/PgJsonB.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Spanner;
+
+use Google\Cloud\Core\JsonTrait;
+use Google\Cloud\Spanner\V1\TypeAnnotationCode;
+use JsonSerializable;
+
+/**
+ * Represents a value with a data type of
+ * [PG JSONB](https://cloud.google.com/spanner/docs/reference/postgresql/data-types) for the
+ * Postgres Dialect database.
+ *
+ * Example:
+ * ```
+ * use Google\Cloud\Spanner\SpannerClient;
+ * 
+ * $spanner = new SpannerClient();
+ * $pgJsonb = $spanner->pgJsonb('{}');
```

Comment:
@dwsupplee 
Probably a nit question, but everywhere we specify `JSONB` as `JsonB`(_for example the class name_). However, here I have used `pgJsonb`, for 2 reasons:
- Consistency with `pgNumeric`.
- It might be easier for the user to use `pgJsonb` instead of `pgJsonB`.

Just wanted to point that out.

Reply:
Great call out - thanks @saranshdhingra. The bullet points seem reasonable to me, but now i'm wondering - should we name the class and other uses `Jsonb`? It seems reasonable as the literal name for the type is `JSONB`, so no need for camelcasing.